Cannot find module 'js-yaml'

```
Error: Cannot find module 'js-yaml'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/var/oscar/web/lib/conf.js:1:74)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

--- a/web/package.json
+++ b/web/package.json
@@ -9,6 +9,7 @@
     "express": "3.4.4",
     "jade": "*",
     "node-trello": "*",
-    "dot": ">=1.0.2"
+    "dot": ">=1.0.2",
+    "js-yaml": "*"
   }
 }

--- a/web/package.json
+++ b/web/package.json
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "express": "3.4.4",
+    "express-dot": "*",
     "jade": "*",
     "node-trello": "*",
     "dot": ">=1.0.2",

